Core/Tests/Speckle.Core.Tests.Unit/Transports/SQLiteTransportTests.cs

```diff
@@ -91,6 +91,42 @@ public void SaveAndRetrieveObject_Sync()
     }
   }
 
+  [Test(
+    Description = "Tests that it is possible to enumerate through all objects of the transport while updating them, without getting stuck in an infinite loop"
+  )]
```

Comment:
Nice ♥️ I'd love you to run me by this just to be 100% sure I'm interpreting this test right